### ISSUE TITLE
[FEATURE] Accessibility

### ISSUE THREAD

**Issue by grantwforsythe** [grantwforsythe/custom-reports-for-ynab#25]
### Describe the feature

Make the application accessible so that it can be used via keyboard inputs and with the use of a screen reader.

### Additional context

Workflows should be implemented to monitor the accessibility of the website.
